cmd/gossamer/README.md

```diff
@@ -0,0 +1,190 @@
+interact with Gossamer to perform administrative tasks such as key management, as well as for interacting with the
+runtime by querying storage and submitting transactions, and inspecting the chain's history.
+
+#### System
+
+The system service is defined in [dot/system/service.go](../../dot/system/service.go) and exposes metadata about the
+Gossamer system, such as the names and versions of the protocols that it implements.
+
+#### Core
+
+As its name implies, the core service ([dot/core/service.go](../../dot/core/service.go)) encapsulates a range of
+capabilities that are central to the functioning of a Gossamer node. In general, the core service is a type of
+dispatcher that coordinates interactions between services, e.g. writing blocks to the database, reloading
+[the runtime](#runtime) when its definition is updated, etc.
+
+### Keystore
+
+The Gossamer keystore ([lib/keystore](../../lib/keystore)) is used for managing the public/private cryptographic key
+pairs that are used for participating in a blockchain network. Public keys are used to identify network participants;
+network participants use their private keys to sign messages in order to authorise privileged actions. In addition to
+informing the Gossamer blockchain client capabilities, the Gossamer keystore is accessible by way of the `account`
+subcommand. The Gossamer keystore manages a number of key types, some of which are listed below:
+
+- `babe` - This key is used for signing messages related to the BABE block production algorithm.
+- `gran` - The GRANDPA key is used for participating in GRANDPA block finalisation.
+- `imon` - The name of this key is a reference to "ImOnline", which is an
+  [online message](https://wiki.polkadot.network/docs/glossary#online-message) that Gossamer nodes use to report
+  liveliness.
+
+### Runtime
+
+In addition to the above-described services, Gossamer hosts a Wasm execution environment that is used to manage an
+upgradeable blockchain runtime. The runtime must be implemented in Wasm, and must expose an interface that is specified
+in [lib/runtime/interface.go](../../lib/runtime/interface.go). The runtime defines the blockchain's state transition
+function, and the various Gossamer services consume this capability in order to author blocks, as well as to verify
+blocks that were authored by network peers. The runtime is dependent on a
+[Wasm host interface](https://docs.wasmer.io/integrations/examples/host-functions), which Gossamer implements and is
+defined in [lib/runtime/wasmer/exports.go](../../lib/runtime/wasmer/exports.go).
+
+### Monitoring
+
+Gossamer publishes telemetry data and also includes an embedded Prometheus server that reports metrics. The metrics
+capabilities are defined in the [dot/metrics](../../dot/metrics) package and build on
+[the metrics library that is included with Go Ethereum](https://github.com/ethereum/go-ethereum/blob/master/metrics/README.md).
```

Comment:
maybe let the user know on which port the prometheus metrics are available.

Reply:
Yes please...do you know the answer to this question?

Reply:
The default port for Prometheus is `9090` so that should be it. It's mentioned in one of the doc, although I don't see it in our code. By the way - OOS - we should have it configurable ideally.

Reply:
there is also a `--metrics-port` flag to set it :)

Reply:
What is the meaning of "OOS" in this context?

Reply:
@noot My bad, awesome! OOS=out of scope 😄 I need to stop using abbreviations I'm sorry!

Reply:
IOKDDWAI

Reply:
(it's okay dude don't worry about it)